Modules/Microsoft365DSC/DSCResources/MSFT_O365OrgCustomizationSetting/readme.md

```diff
@@ -23,7 +23,7 @@ Ensure
 - Description: `Present` is the only value accepted.
   Configurations using `Ensure = 'Absent'` will throw an Error!
 
-GlobalAdminAccount
+Credential
 
 - Required: Yes
 - Description: Credentials of the Office 365 Global Admin
```

Comment:
Same comment we say Exchange Global admin in schema and here we say Office 365 Global admin

Reply:
Done, replaced with Credentials of the account to authenticate with